cms/envs/common.py

```diff
@@ -676,19 +676,20 @@
 
 # Todo: add aws entries for this
 COURSE_UTILITIES = [
-    {"short_description": "Bulk Operations",
-     "items": [
-         {"short_description": "Get all captions from YouTube",
-          "long_description": "This utility will attempt to get or update captions for all videos in the course from YouTube. Please allow it a couple of minutes to run.",
-          "action_url": "utility_captions_handler",
-          "action_text": "Check Captions",
-          "action_external": False},
-#         {"short_description": "Bulk view problem settings",
-#          "long_description": "This utility will allow you to view all section, subsection and problem settings in one page.",
-#          "action_url": "utility_bulksettings_handler",
-#          "action_text": "View Problem Settings",
-#          "action_external" : False}
-          ]
+    {
+        "short_description": "Bulk Operations",
+        "items": [
+            {"short_description": "Get all captions from YouTube",
```

Comment:
This line (after the `{`) should be dropped to a new line. Then, each item in this `dict` should be indented another level.

```python
{
    'short_description': 'Get all captions from YouTube',
}
```